src/contracts/interfaces/IDelegationManager.sol

```diff
@@ -551,6 +563,11 @@ interface IDelegationManager is ISignatureUtils, IDelegationManagerErrors, IDele
     /// @notice Return the M2 minimum withdrawal delay in blocks for backwards compatability
     function minWithdrawalDelayBlocks() external view returns (uint256);
 
+    /// @notice Returns a list of pending queued withdrawals for a `staker`, and the `shares` to be withdrawn.
```

Comment:
Nit: can we add a comment that this is relevant only for post-slashing queued withdrawals?